README.md

```diff
@@ -39,8 +41,12 @@ This will enable the project to access plugin files stored in Maven Central, htt
 
 ### Read & use a BOM
 
-There are different ways to use the plugin depending on your use case.
+There are 3 different ways to use the plugin depending on your use case:
+- [`Bom.dependencies`](#bom-dependencies): for usage in `dependencyOverrides` only
```

Comment:
Anchor should point to an existing section. You can get the actual name of the anchor from the rendered Markdown file. For this section proper anchor is `#use-it-in-dependencyoverrides--bomdependencies`

Reply:
I find them harder to keep in sync: people tend to change titles without noticing it's used as an anchor. But I changed to use the "automatic" anchors then.